package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
Well npm will always install the latests it can so an install will always (at this moment) contain v4.17.4
I don't see an upside to doing the range thing, it will only add maintenance burden to make sure the module works with all the versions